.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: elixir
+elixir:
+  - 1.4.5
+  - 1.5.1
+otp_release:
+  - 19.3
+  - 20.0
```

Comment:
20.1

Reply:
How about we just cover the most current release then? e.g.

 otp_release:
   - 19
   - 20
```

Reply:
sgtm

Reply:
Apparently you can't do that, so I added the numbers back 😄